src/sql/migrations.rs

```diff
@@ -1138,6 +1138,25 @@ CREATE INDEX msgs_status_updates_index2 ON msgs_status_updates (uid);
         .await?;
     }
 
+    inc_and_check(&mut migration_version, 128)?;
+    if dbversion < migration_version {
+        // Add the timestamp of addition and removal.
+        //
+        // If `add_timestamp >= remove_timestamp`,
+        // then the member is currently a member of the chat.
+        // If `add_timestamp < remove_timestamp`,
+        // then the member is a past member.
```

Comment:
Maybe
```suggestion
        // Otherwise the member is a past member.
```
? The reader has to look above to understand what is "otherwise", but when reading the comment from the start it looks simpler.